Framework/DataHandling/test/SaveAscii2Test.h

```diff
@@ -1249,6 +1286,21 @@ class SaveAscii2Test : public CxxTest::TestSuite {
     AnalysisDataService::Instance().add(m_name, wsToSave);
   }
 
+  void write1MDHistoWS(const std::string name) {
+    FrameworkManager::Instance().exec("CreateMDWorkspace", 16, "Dimensions", "3", "Extents", "-5,5,-4,4,-3,3", "Names",
+                                      "H,K,L", "Units", "r.l.u., r.l.u., r.l.u.", "Frames", "HKL,HKL,HKL", "SplitInto",
+                                      "2", "SplitThreshold", "50", "OutputWorkspace", name.c_str());
+
+    FrameworkManager::Instance().exec("FakeMDEventData", 8, "InputWorkspace", name.c_str(), "UniformParams", "100000",
+                                      "PeakParams", "100000,0,0,1,0.3", "RandomSeed", "3873875");
+
+    FrameworkManager::Instance().exec(
+        "BinMD", 18, "InputWorkspace", name.c_str(), "AxisAligned", "0", "BasisVector0",
+        "(0.0+1.0x 0.0 0.0), in 7.26 Ang^-1, 1.0,0.0,0.0", "BasisVector1", "u2, in 7.26 Ang ^ -1, 0.0, 1.0, 0.0",
+        "BasisVector2", "u3, in 3.14 Ang ^ -1,0.0,0.0,1.0", "OutputExtents", "-2.5,2.5,-0.16,0.16,-0.05,0.05",
+        "OutputBins", "50,1,1", "NormalizeBasisVectors", "1", "OutputWorkspace", name.c_str());
```

Comment:
Maybe we could also change this to have `OutputBins = "1,50,1"` as a test that we're getting the non integrated dimension not the first dimension. Sorry only just spotted this!

Reply:
Sorry, do you mean I should add a new header for the Q dimension?